src/ucs/sys/topo/base/topo.c

```diff
@@ -625,3 +625,49 @@ double ucs_topo_get_pci_bw(const char *dev_name, const char *sysfs_path)
     ucs_debug("%s: pci bandwidth undetected, using maximal value", dev_name);
     return DBL_MAX;
 }
+
+const char *ucs_topo_dev_path_to_sysfs_path(const char *dev_path,
+                                            const char *dev_name,
```

Comment:
seems dev_name is used only for debug purpose. can we avoid passing it? print the dev_path in case of failure